Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+click = "*"
+xarray = "*"
+Flask = "*"
+netCDF4 = "*"
+gunicorn = "*"
+
+[dev-packages]
+black = "==21.6b0"
+pytest = "*"
+pre-commit = "*"
+snakeviz = "*"
+
+[requires]
+python_version = "3.8"
+
+[packages.dask]
+extras = [ "dataframe",]
+
+[packages.e1839a8]
+path = "."
+editable = true
```

Comment:
Sweet, apart from the stupid Toml syntax!

Reply:
The nice thing is that we never need to edit this file directly :sunglasses: